CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.2.0 (2021-08-
+------------------
+
+**Added**
+
+* Adds discount percentage and discount per unit to the DTO OfferItem (`#753 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/753>`_)
+
+**Fixed**
+
+**Dependencies**
+
+**Deprecated**
+
```

Comment:
We work on version 1.1.0 afaik 🤔

Reply:
I think you are right, since it was only a release candidate? Shall I add this to the other section, then?

Reply:
I think you dont even need to add it to the CL since it adds fields to a dto which is not officially released yet